packages/lesswrong/lib/rateLimits/constants.ts

```diff
@@ -48,114 +49,127 @@ const LW: {POSTS: PostAutoRateLimit[], COMMENTS: CommentAutoRateLimit[]} = {
   // 2 posts per week rate limits
     {
       ...timeframe('2 Posts per 1 weeks'),
-      karmaThreshold: 4,
       rateLimitType: "newUserDefault",
+      isActive: user => (user.karma < 5),
       rateLimitMessage: `Users with less than 5 karma can write up to 2 posts a week.<br/>${lwDefaultMessage}`,
     }, 
   // 1 post per week rate limits
     {
       ...timeframe('1 Posts per 1 weeks'),
-      karmaThreshold: -1,
+      isActive: user => (user.karma <= -3),
       rateLimitMessage: `Users with -3 or less karma can post once per week.<br/>${lwDefaultMessage}`
     }, 
     {
       ...timeframe('1 Posts per 1 weeks'),
-      last20PostKarmaThreshold: -15,
-      downvoterCountThreshold: 4,
+      isActive: (user, features) => (
+        features?.last20PostKarma <= -15 && 
+        features?.postDownvoterCount >= 4
```

Comment:
I've not gone over everything with a fine-toothed comb, but my guess is that `<=` will be off by one when compared to the old logic (as was implemented in the now-deleted `shouldRateLimitApply` function in `utils.ts`)

Reply:
(could also be the other direction, and it's actually the bare `<`/`>` operators which'll be off by one, but I'm pretty sure that `shouldRateLimitApply` consistently didn't use `or equal to` operators)